mount(8) might not be available in /bin/mount as dmsquash-live-root.sh expects.
It will be in PATH anyway.

--- modules.d/90dmsquash-live/dmsquash-live-root.sh.orig	2012-05-28 15:35:50.791973219 +0200
+++ modules.d/90dmsquash-live/dmsquash-live-root.sh	2012-05-28 15:35:59.742754521 +0200
@@ -204,7 +204,7 @@ if [ -b "$BASE_LOOPDEV" ]; then
     ln -s $BASE_LOOPDEV /run/initramfs/live-baseloop
 fi
 ln -s /dev/mapper/live-rw /dev/root
-printf '/bin/mount %s /dev/mapper/live-rw %s\n' "$ROOTFLAGS" "$NEWROOT" > $hookdir/mount/01-$$-live.sh
+printf 'mount %s /dev/mapper/live-rw %s\n' "$ROOTFLAGS" "$NEWROOT" > $hookdir/mount/01-$$-live.sh
 
 need_shutdown
 
